src/modules/Sharpen/Module.js

```diff
@@ -0,0 +1,46 @@
+/*
+  Sharpen an image
+*/
+module.exports = function Sharpen(options, UI) {
+
+  var defaults = require('./../../util/getDefaults.js')(require('./info.json'));
```

Comment:
Please use `let` instead of `var` wherever possible :)

Reply:
> Also, can you add a test for this module too?

Actually, I am very new to testing but would love to give it a shot if you can guide me how this data file for example got generated. And is there a way to run individual test file.
Every time I try to run a test on a single file using the command `npm test -- SomeTestFileToRun`, all the tests get executed 😅 instead...

<img width="104" alt="Screenshot 2021-03-17 at 2 13 52 PM" src="https://user-images.githubusercontent.com/58583793/111439090-03154080-872b-11eb-92a9-00488394429d.png">

https://github.com/publiclab/image-sequencer/blob/417c1b18261a87acb93ff0d455687e846947add0/test/core/modules/colormap.js#L2-L3

Thanks :)